pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -148,6 +151,12 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.PodInfraContainerImage = imageTemplate.ExpandOrDie("pod")
 	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
 
+	if dockerClient, _, err := docker.NewHelper().GetClient(); err == nil {
```

Comment:
@deads2k how do you feel about making server calls in BuildKubernetesNodeConfig? I'd prefer it not do that, but I also don't want config bits mutating after we build the config here